repose-aggregator/src/docs/asciidoc/filters/attribute-mapping-policy-validation.adoc

```diff
@@ -0,0 +1,58 @@
+= Attribute Mapping Policy Validation Filter
+
+include::../_includes/in-progress.adoc[]
+
+This filter will validate attribute mapping policies in requests to the origin service.
+
+The origin service is expected to be an Identity service.
+This filter will only take action if the HTTP request method is `PUT`.
+
+== General filter information
+* *Name:* attribute-mapping-policy-validation
+* *Released:* TBD
```

Comment:
We probably want to add a task to the next release story to come back and update this.